x-pack/plugins/observability/public/pages/slos/components/slo_list.tsx

```diff
@@ -71,9 +77,23 @@ export function SloList({ autoRefresh }: Props) {
           initialState={state}
         />
       </EuiFlexItem>
-      <EuiFlexItem>
-        <SloListItems sloList={results} loading={isLoading || isRefetching} error={isError} />
+      <EuiFlexItem grow={false}>
+        <ToggleSLOView sloView={sloView} setSLOView={setSLOView} setGridSize={setGridSize} />
```

Comment:
Can I suggest we keep consistent naming for callback props `onChangeSomething={handleChangeSomething}` ? 

Also here, we are storing the grid size in the locale storage, but we are already doing this in the GridSize component. I think we could simplify the GridSize component by always storing the value in the handler here.

```suggestion
        <ToggleSloView sloView={sloView} onChangeView={handleChangeView} onChangeGridSize={handleChangeGridSize} />
```